Sources/FoundationEssentials/Data/Data.swift

```diff
@@ -2848,3 +2948,62 @@ extension Data : Codable {
         }
     }
 }
+
+#if compiler(>=6.2)
+// TODO: remove once _overrideLifetime is public in the standard library
```

Comment:
When will that happen?

Reply:
This is the PR: https://github.com/swiftlang/swift/pull/81225